lib/features/email/presentation/utils/email_utils.dart

```diff
@@ -49,4 +51,32 @@ class EmailUtils {
       return null;
     }
   }
+
+  static bool checkingIfAttachmentActionIsEnabled(Object? state) {
+    if (state is IdleDownloadAttachmentForWeb) {
+      return true;
+    }
+
+    if (state is DownloadAttachmentForWebSuccess) {
+      return true;
+    }
+
+    if (state is DownloadAttachmentForWebFailure) {
+      return true;
+    }
+
+    if (state is IdleViewAttachmentForWeb) {
+      return true;
+    }
+
+    if (state is ViewAttachmentForWebSuccess) {
+      return true;
+    }
+
+    if (state is ViewAttachmentForWebFailure) {
+      return true;
+    }
```

Comment:
Should be use multiple conditions in the same `if`

Reply:
Done